src/com/owncloud/android/files/FileOperationsHelper.java

```diff
@@ -88,13 +93,28 @@ public void openFile(OCFile file) {
             }
             
             Intent chooserIntent;
+            List<ResolveInfo> launchables;
             if (intentForGuessedMimeType != null) {
                 chooserIntent = Intent.createChooser(intentForGuessedMimeType, mFileActivity.getString(R.string.actionbar_open_with));
+                launchables = mFileActivity.getPackageManager().queryIntentActivities(intentForGuessedMimeType, PackageManager.GET_INTENT_FILTERS);
```

Comment:
Please don't introduce new lines with more than 120 columns. That is the maximum number of colums that Github shows in the diff blocks. It is for make easier the code reviews.